sympy/matrices/dense.py

```diff
@@ -1570,12 +1570,21 @@ def casoratian(seqs, n, zero=True):
     return Matrix(k, k, f).det()
 
 
-def randMatrix(r, c=None, min=0, max=99, seed=None, symmetric=False, percent=100):
+def randMatrix(r, c=None, min=0, max=99, seed=None, prng=None, symmetric=False, percent=100):
```

Comment:
The new parameter should be added last, in the (admittedly rare) case that somebody provided parameters by position.